src/Functions/UserDefined/UDFHelper.cpp

```diff
@@ -51,6 +52,9 @@ void validateUDFName(const String & func_name)
 
     if (!std::isalpha(func_name[0]) && func_name[0] != '_')
         throw Exception(ErrorCodes::UDF_INVALID_NAME, "UDF name's first char shall be an alphabetic or underscore");
+
+    if (AggregateFunctionCombinatorPtr combinator = AggregateFunctionCombinatorFactory::instance().tryFindSuffix(func_name))
+        throw Exception(ErrorCodes::UDF_INVALID_NAME, "UDF name can not end up with " + combinator->getName() + ", because it is key word suffix");
```

Comment:
use format `Exception(ErrorCodes::UDF_INVALID_NAME, "UDF name can not end up with {}, because it is key word suffix", combinator->getName());`